tests/e2e/cucumber/steps/ui/adminSettings.ts

```diff
@@ -47,20 +47,50 @@ When(
 )
 
 When(
-  /^"([^"]*)" (disables|deletes) the space "([^"]*)"$/,
+  /^"([^"]*)" (disables|deletes) the space "([^"]*)" using the context-menu$/,
   async function (this: World, stepUser: string, action: string, key: string): Promise<void> {
     const { page } = this.actorsEnvironment.getActor({ key: stepUser })
     const spacesObject = new objects.applicationAdminSettings.Spaces({ page })
 
     switch (action) {
       case 'disables':
-        await spacesObject.disable({ key })
+        await spacesObject.disable({ key, context: 'context-menu' })
         break
       case 'deletes':
-        await spacesObject.delete({ key })
+        await spacesObject.delete({ key, context: 'context-menu' })
         break
       default:
         throw new Error(`${action} not implemented`)
     }
   }
 )
+
+When(
+  /^"([^"]*)" (disables|deletes) the space "([^"]*)" using the batch-actions$/,
```

Comment:
Just to be sure: you still plan to rewrite this to `/^"([^"]*)" (disables|deletes) the following spaces using the batch-actions$/`, right? 🙂

Reply:
yeah right :)